### PR TITLE
UX: improves local-dates form

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: plugins/discourse-local-dates/assets/javascripts/discourse/components/discourse-local-dates-create-form.js.es6 [hunk 1]
```diff
@@ -15,23 +15,154 @@ export default Ember.Component.extend({
...
+  },
+
+  _renderPreview: debounce(function() {
+    const markup = this.get("markup");
+
+    if (markup) {
+      cookAsync(markup).then(result => {
+        this.set("currentPreview", result);
+
+        Ember.run.next(() =>
```
Comment (eviltrout) [inline plugins/discourse-local-dates/assets/javascripts/discourse/components/discourse-local-dates-create-form.js.es6]: I always dislike this kind nested `Ember.run` - is the issue that when the renderPreview is triggered the element it renders into might not exist yet?

File: plugins/discourse-local-dates/assets/javascripts/discourse/templates/components/discourse-local-dates-create-form.hbs [hunk 2]
```diff
@@ -22,20 +39,30 @@
...
             <div class="controls">
-              {{input type="time" value=time class="time-input"}}
+              {{input input=(mut time) type="time" value=time class="time-input"}}
             </div>
           </div>
         </div>
 
-        <span class="to-indicator"><span>{{i18n "discourse_local_dates.create.form.to"}}</span></span>
+        <div class="to-indicator">
+          {{#if site.mobileView}}
```
Comment (eviltrout) [inline plugins/discourse-local-dates/assets/javascripts/discourse/templates/components/discourse-local-dates-create-form.hbs]: This looks like a good candidate for ember's inline-if

**Comment by ZogStriP** [comment]
Where are the before / after screenshots? 😉

File: plugins/discourse-local-dates/assets/javascripts/discourse/templates/components/discourse-local-dates-create-form.hbs (hunk 2 above)
Comment (jjaffeux) [reply plugins/discourse-local-dates/assets/javascripts/discourse/templates/components/discourse-local-dates-create-form.hbs]: indeed, I never use it 👍

File: plugins/discourse-local-dates/assets/javascripts/discourse/components/discourse-local-dates-create-form.js.es6 (hunk 1 above)
Comment (jjaffeux) [reply plugins/discourse-local-dates/assets/javascripts/discourse/components/discourse-local-dates-create-form.js.es6]: I think this is not needed anymore, it's the result of a refactoring where I have removed something in between

**Comment by jjaffeux** [comment]
> Where are the before / after screenshots? 😉

well in the dedicated topic :p

**Pull request merged**